tests/explainers/test_tree.py

```diff
@@ -391,8 +391,13 @@ def test_gpboost():
 def test_catboost():
     catboost = pytest.importorskip("catboost")
     # train catboost model
+<<<<<<< HEAD
+    X, y = shap.datasets.boston()
+    X["RAD"] = X["RAD"].astype(int)
+=======
     X, y = shap.datasets.california(n_points=500)
     X['IsOld'] = (X['HouseAge'] > 30).astype(str)
+>>>>>>> 45b85c1837283fdaeed7440ec6365a886af4a333
```

Comment:
There is an open merge conflict here.

Reply:
Oops, thanks!

Reply:
Found an unnecessary dot here
https://github.com/burk/shap/commit/be2658e62df109e0d66b448652d7f0b8cb05619f